seirmo/apps/simulation_app.py

```diff
@@ -0,0 +1,76 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import pandas as pd
```

Comment:
Packages should be ordered in 

<native> (os, unittest, and so on)

<external> (numpy, pandas, dash and so on)

<internal> (in this case seirmo)